index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Emma's Bio</title>
+  <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+<body>
+
+  <header class="Header">
+    <h1 class="Header__headline">Well hello there! I’m Emma and  I’m out of work</h1>
+  </header><!-- Header -->
```

Comment:
Good use of comments to indicate the end of a block.  👍   Well commented code is good code.  Definitely use this technique through out your page.